packages/wonder-blocks-modal/util/maybe-get-portal-mounted-modal-host-element.js

```diff
@@ -0,0 +1,44 @@
+// @flow
+import {ModalLauncherPortalAttributeName} from "./constants.js";
+
+/**
+ * From a given element, finds its next ancestor that is a modal launcher portal
+ * element.
+ * @param {null | Element | Text} element The element whose ancestors are to be
```

Comment:
🤷‍♂️ These comments are nice but I don't think we need to use the full `@param` & `@returns` syntax (we're not doing anything with it and I think the Flow types themselves are sufficient).

Reply:
It's the document extension in code. I type /** return and it adds almost all the boilerplate. Happy to get rid of it, but it's really nice in the intellisense of VSCode (at least I think so).